src/libs/actions/Search.ts

```diff
@@ -403,14 +413,14 @@ function payMoneyRequestOnSearch(hash: number, paymentData: PaymentData[], trans
     ];
 
     const optimisticData: OnyxUpdate[] = createOnyxData({isActionLoading: true});
-    const failureData: OnyxUpdate[] = createOnyxData({errors: getMicroSecondOnyxErrorWithTranslationKey('common.genericErrorMessage')});
-    const finallyData: OnyxUpdate[] = createOnyxData({isActionLoading: false});
```

Comment:
We should set `isActionLoading` back to false if the current search key is not pay (meaning that we still have the object in onyx).

(this should be fixed for other optimistic actions too)